README.md

```diff
@@ -1,12 +1,12 @@
-# Kubermatic Documentation
+# Kubermatic Products Documentation
 
-Kubermatic is a Cluster-as-a-Service that provides managed Kubernetes for your infrastructure.
+Kubermatic products consisting of Kubermatic Kubernetes Platform, KubeOne and Kubecarrier are Cluster-as-a-Service that provides managed Kubernetes for your infrastructure.
```

Comment:
kkp - cluster aaS; kubeone - manage highly available k8s clusters; KubeCarrier - Application Mngt for thousands of apps